client/post-editor/controller.js

```diff
@@ -131,10 +159,13 @@ module.exports = {
 			// We have everything we need to start loading the post for editing,
 			// so kick it off here to minimize time spent waiting for it to load
 			// in the view components
-			if ( postID ) {
+			if ( postID && ! copyId ) {
 				// TODO: REDUX - remove flux actions when whole post-editor is reduxified
 				actions.startEditingExisting( siteId, postID );
 				analytics.pageView.record( '/' + postType + '/:blogid/:postid', gaTitle + ' > Edit' );
+			} else if ( copyId ) {
+				startEditingPostCopy( siteId, copyId );
```

Comment:
Why isn’t `startEditingPostCopy` a new action like the others, but a function in the controller?